packages/ui/src/routes/bridge/BridgeConfirmPage.tsx

```diff
@@ -229,6 +237,27 @@ const BridgeConfirmPage: FunctionComponent<{}> = () => {
         ? hasNativeAssetBalance
         : hasNativeAssetBalance && hasFromTokenBalance
 
+    const nativeTokensInDestinationNetworkStatus =
+        useAddressHasEnoughNativeTokensToSend(
+            bridgeQuote.bridgeParams.params.toChainId,
+            bridgeQuote.bridgeParams.params.toToken.address
+        )
+
+    const destinationNetwork = Object.values(availableNetworks).find(
+        (n) => n.chainId === bridgeQuote.bridgeParams.params.toChainId
+    )
+    const bridgeWarningMessage = getWarningMessages(
+        nativeTokensInDestinationNetworkStatus.result,
+        destinationNetwork
+    )
+
+    const showDestinationFeeWarning =
```

Comment:
can we add a comment here explaining the cases where the warning will be displayed?